simtools/utils/names.py

```diff
@@ -39,10 +39,23 @@ def array_elements():
         return yaml.safe_load(file)["data"]
 
 
-site_names = {
-    "South": ["paranal", "south", "cta-south", "ctao-south", "s"],
-    "North": ["lapalma", "north", "cta-north", "ctao-north", "n"],
-}
+@lru_cache(maxsize=None)
+def site_names():
+    """
+    Site names from reference file.
+
+    The list of sites is derived from location of available array elements
+    Return a dictionary for compatibility with the validation routines.
```

Comment:
I found this explanation a bit confusing. Could you please improve it a bit?
E.g. "The list of sites is derived from the array element schema file."
"Returns a dictionary that is compatible with..."

Reply:
Good point, improved it.